components/brave_new_tab_ui/brave_new_tab_page.mojom

```diff
@@ -49,7 +49,7 @@ interface PageHandlerFactory {
 struct SearchEngineInfo {
   string name;
   string keyword;
-  string origin;
+  string host;
```

Comment:
Aha, `origin` is where current page is from, so `host` is better in this context, isn't it?

Reply:
I switched to host because that's what the other functions in `TemplateURLService` use, and it can be a hassle converting :laugh

Reply:
Yeah I can see that